app/src/main/res/drawable/rounded_outline.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+    android:color="@android:color/white">
+
+    <item>
+        <shape android:shape="oval">
+            <stroke
+                android:width="2dp"
+                android:color="?attr/white" />
+            <corners android:radius="@dimen/rounded_image_radius" />
```

Comment:
This is wrong, the cardCornerRadius is set to 25dp and while this MIGHT look fine, but rounded_image_radius may be subject to change which makes this a hard to track bug. (Also android on lower vers have difficulties with corners in some edge cases so this must match exactly)